recipes/gsoap/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "gsoap" %}
+{% set version = "2.8.78" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/gsoap2/gsoap-2.8/gsoap_{{ version }}.zip
```

Comment:
The hardcoded part of the version in the URL will break the updating bot. You need to use a `split` in the jinja variable to get only the `2.8` here.